init_features.go

```diff
@@ -3,57 +3,78 @@ package chalk
 import (
 	"fmt"
 	"github.com/chalk-ai/chalk-go/internal"
-	"github.com/pkg/errors"
+	"github.com/cockroachdb/errors"
```

Comment:
Is this change intentional?

Reply:
Yes - I'm trying to phase out the usage of pkg/errors in favor of the cockroach one